relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -75,10 +75,11 @@ fn get_trace_context(event: &Event) -> Option<&TraceContext> {
     None
 }
 
+/// Extract transaction status, defaulting to [`SpanStatus::Unknown`].
+/// Must be consistent with `process_trace_context` in [`relay_general::store::normalize`].
 #[cfg(feature = "processing")]
-fn extract_transaction_status(trace_context: &TraceContext) -> Option<String> {
-    let span_status = trace_context.status.value()?;
-    Some(span_status.to_string())
+fn extract_transaction_status(trace_context: &TraceContext) -> SpanStatus {
+    *trace_context.status.value().unwrap_or(&SpanStatus::Unknown)
```

Comment:
For reference: 
https://github.com/getsentry/relay/blob/d03b65bff49744aad1abc54d7ff10d53d20a8d3e/relay-general/src/store/normalize.rs#L700-L711